dvc/command/config.py

```diff
@@ -136,6 +135,27 @@ def _check(self, conf, remote, section, opt=None):
                 f"option '{opt}' doesn't exist in {name} '{section}'"
             )
 
+    def _get_appropriate_levels(self, levels):
+        if levels:
+            self._validate_level_for_non_repo_operation(levels)
+            return [self.args.level]
+        if self.config.dvc_dir is None:
+            return [
+                level
+                for level in self.config.LEVELS
+                if level not in self.config.REPO_ONLY_LEVELS
+            ]
+        return self.config.LEVELS
```

Comment:
If you see my other suggestion, we could just do

```suggestion
            return self.config.SYSTEM_LEVELS
        return self.config.LEVELS
```